### DISCUSSION TITLE
PCB Manufacturing and Assembly

### DISCUSSION THREAD

**Discussion by nnickthomas** [meganetaaan/stack-chan#128] [Q&A]
Hello, 

When getting a stackchan board made and assembled from JLCPCB, do any settings have to be specifically changed?  I am hoping to build a version with SG90 servos and I understand the [Parts List](https://github.com/meganetaaan/stack-chan/tree/dev/v1.0/schematics#parts-list) contains the required components for both the SG90 version and the RS304MD version.  

Thank you!

**Comment by meganetaaan** [comment]
Apologies for the delayed response!

Please note that the BOM and placement settings provided in the repository are designed for serial servos (RS30X, SCS0009). If you want to assemble the board with SG90 servos, you will need to create your own BOM and placement settings by following the assembly instructions provided.

Assembly Instructions for PWM Servo: https://github.com/meganetaaan/stack-chan/tree/dev/v1.0/schematics#pwm-servo
Instructions on how to generate BOM and placement settings using KiCAD: https://support.jlcpcb.com/article/84-how-to-generate-the-bom-and-centroid-file-from-kicad

If you have any questions or need further clarification, please feel free to ask on this thread. I am also available on Discord: https://discord.com/invite/b2zYSezuHa.
